pagantis/pagantis.php

```diff
@@ -96,14 +102,14 @@ function plgVmConfirmedOrder ($cart, $order) {
         $jinput = JFactory::getApplication()->input;
 
         //Account Settings
-        $environment = $method->pagamastarde_env;
-        $discount = $method->pagamastarde_discount;
+        $environment = $method->pagantis_env;
+        $discount = $method->pagantis_discount;
         if($environment == 'test'){
```

Comment:
Improve the admin panel to show only a field for the public_key and other one for secret_key, like we are doing in the rest of the modules

Reply:
Done